packages/services/Team/lib/TeamService.js

```diff
@@ -70,6 +72,16 @@ class TeamService extends Singleton {
 
             txBuilder.addCmd(createAccountCmd);
             entityId = createAccountCmd.getProtocolEntityId();
+
+            if (isCreateDefaultProject) {
+              const createProjectCmd = new CreateProjectCmd({
+                teamId: entityId,
+                description: crypto.hexify(crypto.sha256(new TextEncoder('utf-8').encode(JSON.stringify([])).buffer)),
+                domains: ['2519ef55e1b69f1a7e13275e3273950cce7e26a8'],
```

Comment:
Replace it with "7c3d37cbfea2513a7e03e674448bbeee8ae3d862" and leave a comment that this is temporal placeholder